hash/hash.go

```diff
@@ -140,7 +140,7 @@ func ChooseSubset(from sets.String, n int, target string) sets.String {
 	//    2.1. While that index is already selected pick next index
 	// 3. Advance angle by `step`
 	// 4. Goto 1.
-	selection := sets.NewInt()
+	selection := sets.Set[int]{}
```

Comment:
specific reason for not using `sets.New[int]` here? (to keep consistency with other changes)

Reply:
Fixed.